spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -16,6 +17,47 @@
     assert_response :redirect, :action => 'index'
   end
 
+  describe "test_new" do
+    before(:each) do
+      get :new, :format => 'html'
+    end
+
+    it 'should render template new' do
+      assert_template 'new'
+      assert_tag :tag => "table",
+        :attributes => { :id => "category_container" }
+    end
+
+    it 'should have a new category' do
```

Comment:
This test is checking that the @category instance variable is set in the controller to a new empty category that does not have any information saved to it yet. I was modeling a similar test for the category edit to check for a valid category. For the new method @category will be a new category with no information saved yet, and for the edit method it will be the category that is being changed.